src/containers/pages/ServerSettings/EditSettings/tests/index.test.tsx

```diff
@@ -23,6 +25,8 @@ const history = createBrowserHistory();
 /** register the reducers */
 reducerRegistry.register(settingsReducerName, settingsReducer);
 reducerRegistry.register(locationReducerName, locationsReducer);
+store.dispatch(fetchLocSettings(allSettings, '75af7700-a6f2-448c-a17d-816261a7749a'));
+store.dispatch(fetchLocs(locHierarchy));
```

Comment:
were this left out before, what necessitated this?

Reply:
I think using different version of `@opensrp/server-service` in the web and @opensrp/population-characteristics affected mocking of `OpenSRPService` hence the need to do the dispatch manually.